lms/envs/common.py

```diff
@@ -2582,9 +2582,8 @@
 # Enrollment API Cache Timeout
 ENROLLMENT_COURSE_DETAILS_CACHE_TIMEOUT = 60
 
-# for Student Notes we would like to avoid too frequent token refreshes (default is 30 seconds)
-if FEATURES['ENABLE_EDXNOTES']:
-    OAUTH_ID_TOKEN_EXPIRATION = 60 * 60
+
+OAUTH_ID_TOKEN_EXPIRATION = 60 * 60
```

Comment:
@nasthagiri As we got green build and i'd like to merge this without this comment because for another push we have to wait for 1 more hour. This can be added in later PRs. What do you suggest ?